src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -1090,6 +1092,26 @@ cfg_if! {
 pub const PTHREAD_MUTEX_ADAPTIVE_NP: ::c_int = 3;
 
 extern "C" {
+    pub fn fgetspent_r(
+        fp: *mut ::FILE,
+        spbuf: *mut spwd,
```

Comment:
```suggestion
        spbuf: *mut ::spwd,
```

You should use `spwd` like this not to import like above. The following are the same.

Reply:
Neat!